packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8731,627 +8782,7 @@ describe('plugin-meetings', () => {
-        });
-
-        forEach(
-          [
-            {
-              meetingInfo: {},
-              selfUserPolicies: {
-                [SELF_POLICY.SUPPORT_VIDEO]: true,
-              },
-              expectedActions: {
-                canDoVideo: true,
-              },
-            },
-            {
-              meetingInfo: {},
-              selfUserPolicies: {
-                [SELF_POLICY.SUPPORT_VIDEO]: false,
-              },
-              expectedActions: {
-                canDoVideo: true,
-              },
-            },
-            {
-              meetingInfo: {some: 'data'},
-              selfUserPolicies: {
-                [SELF_POLICY.SUPPORT_VIDEO]: true,
-              },
-              expectedActions: {
-                canDoVideo: false,
-              },
-            },
-            {
-              meetingInfo: {some: 'data'},
-              selfUserPolicies: undefined,
-              expectedActions: {
-                canDoVideo: true,
-              },
-            },
-            {
-              meetingInfo: {
-                video: {},
-              },
-              selfUserPolicies: {
-                [SELF_POLICY.SUPPORT_VIDEO]: true,
-              },
-              expectedActions: {
-                canDoVideo: true,
-              },
-            },
-            {
-              meetingInfo: undefined,
-              selfUserPolicies: {},
-              expectedActions: {
-                canDoVideo: true,
-              },
-            },
-            {
-              meetingInfo: {
-                video: {},
-              },
-              selfUserPolicies: {
-                [SELF_POLICY.SUPPORT_VIDEO]: false,
-              },
-              expectedActions: {
-                canDoVideo: false,
-              },
-            },
-          ],
-          ({meetingInfo, selfUserPolicies, expectedActions}) => {
-            it(`has expectedActions ${JSON.stringify(
-              expectedActions
-            )} when policies are ${JSON.stringify(
-              selfUserPolicies
-            )} and meetingInfo is ${JSON.stringify(meetingInfo)}`, () => {
-              meeting.meetingInfo = meetingInfo;
-              meeting.selfUserPolicies = selfUserPolicies;
-              meeting.config.experimental.enableUnifiedMeetings = true;
-
-              meeting.updateMeetingActions();
-
-              assert.deepEqual(
-                {
-                  canDoVideo: meeting.inMeetingActions.canDoVideo,
-                },
-                expectedActions
-              );
-            });
-          }
-        );
-
-        it('correctly updates the meeting actions', () => {
-          // Due to import tree issues, hasHints must be stubed within the scope of the `it`.
-          const restorableHasHints = ControlsOptionsUtil.hasHints;
-          ControlsOptionsUtil.hasHints = sinon.stub().returns(true);
-          ControlsOptionsUtil.hasPolicies = sinon.stub().returns(true);
```

Comment:
Is this by mistake or we do not need these codes? That's some big bulk removal there?

Reply:
This code is removed because this is duplicate code. Check in beta branch also these tests are only present once but in next they were present twice. They are also running the same tests.